junebug/channel.py

```diff
@@ -29,6 +30,13 @@ class InvalidChannelType(JunebugError):
     'xmpp': 'vumi.transports.xmpp.XMPPTransport',
 }
 
+allowed_message_fields = [
+    'transport_name', 'timestamp', 'in_reply_to', 'to_addr', 'from_addr',
+    'content', 'session_event', 'helper_metadata', 'message_id']
+# excluded fields: from_addr_type, group, provider, routing_metadata,
+# to_addr_type, from_addr_type, message_version, transport_metadata,
+# message_type, transport_type
```

Comment:
The `allowed_message_fields` is actually just a list of things that we return on the http request, I should probably find a better name for it. This is just a rough draft of the list, which is why I put the comment of the excluded fields, so that we don't forget about them. The idea is to reveal as little as possible on the API, to make it simpler to understand.